### PR TITLE
Add command to output validator set

### LINKED ISSUE #1048: CLI users SBAT see the current validator set
### Expected Behavior

There's a CLI command to get the list of current validators.

### PR DESCRIPTION
**Pull request by m-chrzan**
### Description

`celocli validatorset` will print the addresses of the current validator set.

### Tested

Against a dev testnet with most recent celo-blockchain and contracts.

### Other changes

Added `validatorSetAddresses` to contractkit wrapper.

### Related issues

- Fixes #1048

### CODE REVIEW

File: packages/cli/src/commands/validatorset.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  static description = 'Outputs the current validator set'
+
+  static flags = {
+    ...BaseCommand.flags,
+  }
+
+  static examples = ['validatorset']
+
+  async run() {
+    const validators = await this.kit.contracts.getValidators()
```
Comment (mcortesi) [inline packages/cli/src/commands/validatorset.ts:13]: I think this is a great case for having `getValidatorsSet()` in the Validators Wrapper.

having a `getLength` + `getElementWithIdx()` sound too loo level for the wrapper

**Review by mcortesi — CHANGES_REQUESTED** [review]

File: packages/contractkit/src/wrappers/Validators.ts [hunk 2]
```diff
@@ -48,6 +48,19 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
     toNumber
   )
 
+  async validatorSetAddresses(): Promise<string[]> {
```
Comment (mcortesi) [inline packages/contractkit/src/wrappers/Validators.ts]: Not sure here... should we use a `getXXX()`  convention? or leave it as it is.

I leave that to you

File: packages/contractkit/src/wrappers/Validators.ts [hunk 3]
```diff
@@ -48,6 +48,19 @@ export class ValidatorsWrapper extends BaseWrapper<Validators> {
...
+    const numberValidators = await this.numberValidatorsInCurrentSet()
+
+    const validators = []
+
+    for (let i = 0; i < numberValidators; i++) {
```
Comment (mcortesi) [inline packages/contractkit/src/wrappers/Validators.ts]: If you want to make @nambrot happy... you can do

```ts
const validators = await Promise.all(
  Array.from(Array(numberValidators).keys()).map(i => this.validatorAddressFromCurrentSet(i))
)
```

and run calls in parallel

**Review by mcortesi — APPROVED** [review]

File: packages/contractkit/src/wrappers/Validators.ts (hunk 3 above)
Comment (mcortesi) [reply packages/contractkit/src/wrappers/Validators.ts]: The "range" array comes from https://stackoverflow.com/questions/3746725/how-to-create-an-array-containing-1-n?page=1&tab=votes#tab-top

and it's a good fn for utils :P

Comment (m-chrzan) [reply packages/contractkit/src/wrappers/Validators.ts]: So the whole `Array.from...` thing for creating a range in js is kind of unfortunate, think a for loop is the lesser evil. But definitely can `Promise.all` this stuff!

File: packages/contractkit/src/wrappers/Validators.ts (hunk 2 above)
Comment (m-chrzan) [reply packages/contractkit/src/wrappers/Validators.ts]: Let's keep with the convention.

**Pull request merged**